packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-entity-type-list.tsx

```diff
@@ -163,7 +162,10 @@ export const AccountEntityTypeList: FunctionComponent<
 
   const accountEntityTypes = useMemo(() => {
     if (allEntityTypes) {
-      const ownedById = getOwnedById(activeWorkspace);
+      const ownedById =
```

Comment:
Can you actually change this function to take `activeWorkspaceAccountId`? It's available and passed in to a sibling component in the parent

Reply:
Addressed in https://github.com/hashintel/hash/pull/1596/commits/32a664811f4059a8d9129a453e6161edb8cdc987

Reply:
Addressed in https://github.com/hashintel/hash/pull/1596/commits/32a664811f4059a8d9129a453e6161edb8cdc987